main.py

```diff
@@ -263,10 +263,12 @@ def ui_access_wrapper(self, *args, **kwargs):
         r'%s/<comma_separated_skill_ids>' % feconf.QUESTIONS_LIST_URL_PREFIX,
         questions_list.QuestionsListHandler),
     get_redirect_route(
-        r'%s/<topic_name>' % feconf.PRACTICE_SESSION_URL_PREFIX,
+        r'%s/<topic_name>' %
+        feconf.PRACTICE_SESSION_URL_PREFIX,
```

Comment:
Any specific reasonf or this change?

Reply:
Oh, no. I had added another field in the middle, but removed that later on, will revert this back.

Reply:
@shavavo has reverted these in #9397